app/uk/gov/hmrc/gform/translation/TranslationController.scala

```diff
@@ -90,6 +104,13 @@ class TranslationController(
       ) { csv =>
         formTemplateService
           .get(FormTemplateRawId(formTemplateId.value))
+          .map(r => FormTemplateRaw(r.value + ("languages" -> Json.toJson(Seq("en", "cy")))))
+          .flatMap(t =>
+            handler.handleRequest(t).value.flatMap {
+              case Right(_)              => Future.successful(t)
+              case Left(unexpectedState) => Future.failed(new RuntimeException(unexpectedState.error))
+            }
+          )
```

Comment:
Why are we inserting json twice, with two different implementation? Why not do it only once?
```
interpreter
              .handleRequest(FormTemplateRaw(jsonToSave))```